src/main/scala/com/amazon/deequ/repository/AnalysisResultSerde.scala

```diff
@@ -513,7 +513,7 @@ private[deequ] object MetricSerializer extends JsonSerializer[Metric[_]] {
         result.addProperty("instance", doubleMetric.instance)
         result.addProperty("name", doubleMetric.name)
         result.addProperty("value", doubleMetric.value.getOrElse(null).asInstanceOf[Double])
-        doubleMetric.fullColumn.foreach(c => result.addProperty("fullColumn", c.expr.sql))
+        // doubleMetric.fullColumn.foreach(c => result.addProperty("fullColumn", c.expr.sql))
```

Comment:
Maybe we can create an issue in Github as well and link it here?

Reply:
done